### DISCUSSION TITLE
Oxia Java SDK

### DISCUSSION THREAD

**Discussion by suyambutriveni** [oxia-db/oxia#346] [Q&A] [answered]
Hi @merlimat ,
In your pulsar summit talk about Oxia, you mentioned about Oxia client SDKs are available for Java and Go. Is the Java SDK available for public?

**Comment by merlimat** [comment, marked as answer]
@suyambuganesh82 The code for the Java SDK is in a different repository and it will be open-sourced ASAP.

**Reply by suyambutriveni** [reply]
Thank you for the answer @merlimat

**Reply by suyambutriveni** [reply]
@merlimat Is Oxia being used in production environment or is it still in early stages. Also is Java SDK for Oxia available?

**Comment by merlimat** [comment]
Oxia Java and Pulsar integration is now available: https://github.com/streamnative/oxia-java

**Reply by hpvd** [reply]
awesome! Many thanks for all the work!
Maybe you also want to give a short hint in this discussion:
https://github.com/apache/pulsar/discussions/21039

**Reply by suyambutriveni** [reply]
Thank you.
